### PR TITLE
16-comments-list-api

### PR DESCRIPTION
**Pull request by W-Shih**
- feature: add comments-list-api
- test: add tests for comments-list-api
- lint: Enable wrong-import-order

### CODE REVIEW

File: comments/api/tests.py [hunk 1]
```diff
@@ -123,8 +124,8 @@ def test_update_api(self):
         # <Wayne Shih> 13-Nov-2021
         # TODO:
         #   This check might be not not suitable when retrieve api is considered.
-        response = self.anonymous_client.get(url)
-        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+        response = self.anonymous_client.patch(url)
+        self.assertEqual(response.status_code, status.HTTP_405_METHOD_NOT_ALLOWED)
```
Comment (wen-wayne-shih) [inline comments/api/tests.py:128]: Remove TODO

File: comments/api/tests.py [hunk 2]
```diff
@@ -200,8 +201,8 @@ def test_delete_api(self):
...
```
Comment (wen-wayne-shih) [inline comments/api/tests.py:205]: Remove TODO

File: comments/api/views.py [hunk 3]
```diff
@@ -35,7 +36,30 @@ class CommentViewSet(viewsets.GenericViewSet):
     # <Wayne Shih> 13-Nov-2021
     # TODO:
     #   This permission_classes might be not not suitable when retrieve api is considered.
-    permission_classes = [IsAuthenticated, IsObjectOwner]
+    # permission_classes = [IsAuthenticated, IsObjectOwner]
```
Comment (wen-wayne-shih) [inline comments/api/views.py:39]: Remove TODO

**Review by wen-wayne-shih — CHANGES_REQUESTED** [review]
address before merging

File: comments/api/views.py (hunk 3 above)
Comment (W-Shih) [reply comments/api/views.py]: Resolved as per suggestion

File: comments/api/tests.py (hunk 2 above)
Comment (W-Shih) [reply comments/api/tests.py]: Resolved as per suggestion

File: comments/api/tests.py (hunk 1 above)
Comment (W-Shih) [reply comments/api/tests.py]: Resolved as per suggestion

**Review by wen-wayne-shih — CHANGES_REQUESTED** [review]
Squash commits before merging

**Review by wen-wayne-shih — APPROVED** [review]

**Pull request merged**